spec/features/idv/in_person_spec.rb

```diff
@@ -507,7 +507,28 @@
       # prepare page
       complete_prepare_step(user)
 
-      complete_state_id_step(user, same_address_as_id: false, include_address: true)
+      complete_state_id_step(user, same_address_as_id: false, double_address_verification: true)
+
+      complete_address_step(user, double_address_verification: true)
+    end
+
+    context 'when users select Puerto Rico' do
+      it 'displays hint text' do
+        sign_in_and_2fa_user(user)
+        begin_in_person_proofing(user)
+        search_for_post_office
+
+        # location page
+        location = page.find_all('.location-collection-item')[1]
```

Comment:
test is failing here

Reply:
Test has been updated to use a more stable version of this step.